soh/src/overlays/actors/ovl_En_Gm/z_en_gm.c

```diff
@@ -260,7 +273,16 @@ void func_80A3DF00(EnGm* this, GlobalContext* globalCtx) {
         this->actor.parent = NULL;
         this->actionFunc = func_80A3DF60;
     } else {
-        func_8002F434(&this->actor, globalCtx, GI_SWORD_KNIFE, 415.0f, 10.0f);
+        if (gSaveContext.n64ddFlag && Randomizer_GetSettingValue(RSK_SHUFFLE_MERCHANTS) &&
+        !Flags_GetRandomizerInf(RAND_INF_MERCHANTS_MEDIGORON) {
```

Comment:
This should fix the build error @stratomaster64 
```suggestion
        !Flags_GetRandomizerInf(RAND_INF_MERCHANTS_MEDIGORON)) {
```